lisa/tests/kernel/scheduler/eas_behaviour.py

```diff
@@ -372,6 +373,41 @@ def test_task_placement(self, energy_est_threshold_pct=5, nrg_model:EnergyModel=
         res.add_metric("energy threshold", threshold, 'bogo-joules')
         return res
 
+    def test_slack(self, negative_slack_allowed_pct=15) -> ResultBundle:
```

Comment:
Agreed with the change, we might want to have that in a RTATestBundle helper/decorator class but since it's the only user for now we can leave it as is.

Reply:
actually, we can keep it in RTATestBundle if it has any value there, and just not annotate it. Then we put an annotated stub in eas_behavior and off we go